sefaria/model/topic.py

```diff
@@ -71,9 +100,9 @@ def _validate(self):
         super(Topic, self)._validate()
         if getattr(self, 'subclass', False):
             assert self.subclass in self.subclass_map, f"Field `subclass` set to {self.subclass} which is not one of the valid subclass keys in `Topic.subclass_map`. Valid keys are {', '.join(self.subclass_map.keys())}"
-
-        if getattr(self, 'portal_slug', None):
```

Comment:
Why were these lines deleted?

Reply:
I mistakenly committed when I copied over the code from the portal to `topic.py` for the sake of mimicking the pattern - needed to delete those lines.